scripts/data_transfer/copy_chr22_skat_input.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gsutil -m cp gs://cpg-tob-wgs-main/CellRegMap_input_files/plink_files/plink_chr22.bed \
+    gs://cpg-tob-wgs-main/CellRegMap_input_files/plink_files/plink_chr22.bim \
+    gs://cpg-tob-wgs-main/CellRegMap_input_files/plink_files/plink_chr22.fam \
+    gs://cpg-tob-wgs-main/CellRegMap_input_files/GRM/grm_wide.csv \
+    gs://cpg-tob-wgs-main/CellRegMap_input_files/expression_objects/sce22.h5ad gs://cpg-tob-wgs-test/v0/skat/
```

Comment:
We usually group things by folder, followed by version -- did you mean `skat/v0` instead of `v0/skat`?

Reply:
Adding onto this, should they appear in similar places on main / test?